src/anders_mathiesen_ex/ex01/letter_counts.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+
+
+def letter_freq(txt):
+    frequencies = {}
+    for symbol in list(txt):
```

Comment:
this is not case-insensitive, i.e. 
`In [2]: letter_freq("HHhhhh")                                                      
Out[2]: {'H': 2, 'h': 4}`